src/libs/ErrorUtils.js

```diff
@@ -37,6 +38,16 @@ function getAuthenticateErrorMessage(response) {
     }
 }
 
+/**
+ * Method used to generate errors object optimistically.
```

Comment:
Once we rename the method i think this description could be updated.
```suggestion
 * Method used to get an error object with microsecond as the key.
```